src/protractor.conf.js

```diff
@@ -34,6 +36,23 @@ const chromeConfig = {
   }
 };
 
+const firefoxConfig = {
+  browserName: 'firefox',
+  marionette: true,
+  'moz:firefoxOptions': {
+    args: []
+  }
+};
+
+const configureBrowsers = async () => {
```

Comment:
It's not entirely what I expected. In my opinion we should support `--chrome` too here and build an array of configurations here instead of returning single item array